hawkular-wildfly-agent-feature-pack/src/main/resources/configuration/standalone/template.xml

```diff
@@ -18,4 +18,85 @@
 
 -->
 <server xmlns="urn:jboss:domain:4.0">
+
+  <extensions>
+        <?EXTENSIONS?>
+  </extensions>
+
+  <management>
```

Comment:
I have no attempted to run on Wildfly-Core. I don't even know where that project is at the moment (is that still going to be released now that we have swarm?). But back in the day, the talk was we want hawkular agent to be small enough and not have large amounts of dependencies so it can run inside of WildFly-Core. This was the initial thinking of Heiko Braun when he wrote the first agent prototype - whether it is something we still want to be able to support is something for @theute and @pilhuhn to confirm. I always assumed we would eventually want to install agents inside of WildFly-Core and swarm-based containers that strip out JEE stuff. I wonder if @bobmcwhirter has tried installing agents in smaller swarm containers?